modules/mesaides/templates/nginx_config.erb

```diff
@@ -0,0 +1,56 @@
+# MANAGED BY PUPPET
+server {
+  listen                *:80 default_server;
+  server_name           <%= @name %>;
+
+  access_log            /var/log/nginx/<%= @name %>.access.log combined;
+  error_log             /var/log/nginx/<%= @name %>.error.log;
+
+  location /.well-known {
+    root                <%= @webroot_path %>;
+  }
+<% if @use_ssl and File.exists?("/etc/letsencrypt/live/#{@name}/fullchain.pem") -%>
+  location / {
+    return              302 https://<%= @name %>$request_uri;
+  }
+<%- else -%>
+  location / {
+    proxy_pass          http://localhost:8000;
+    proxy_redirect      off;
+  }
+<% end -%>
+}
+
+<% if @use_ssl and File.exists?("/etc/letsencrypt/live/#{@name}/fullchain.pem") -%>
+server {
+    listen              443 ssl;
+    server_name         <%= @name %>;
+
+    ssl_certificate     /etc/letsencrypt/live/<%= @name %>/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/<%= @name %>/privkey.pem;
+
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers EECDH+ECDSA+AESGCM:EECDH+aRSA+AESGCM:EECDH+ECDSA+SHA512:EECDH+ECDSA+SHA384:EECDH+ECDSA+SHA256:EDH+aRSA:EECDH:!aNULL:!eNULL:!LOW:!RC4:!3DES:!MD5:!EXP:!PSK:!SRP:!DSS;
```

Comment:
@MattiSG Mozilla's ciphersuite is not great. SSLLabs gives an F to mes-aides.gouv.fr.
https://verif.site/#mes-aides.gouv.fr